rhamt/tests/test_projects.py

```diff
@@ -0,0 +1,14 @@
+import fauxfactory
+
+
+def test_project(application):
+    project_name = fauxfactory.gen_alphanumeric(start="project_")
+    project_collection = application.collections.projects
+    # TO DO paramterize the test later for file_name and trans_path, hardcoding for now
+    project_collection.create(
+        name=project_name,
+        description=fauxfactory.gen_alphanumeric(),
+        file_name="acmeair-webapp-1.0-SNAPSHOT.war",
+        transformation_path="Containerization",
+    )
+    assert project_collection.exists(project_name)
```

Comment:
This will become
```
proj =     project_collection.create(
        name=project_name,
        description=fauxfactory.gen_alphanumeric(),
        file_name="acmeair-webapp-1.0-SNAPSHOT.war",
        transformation_path="Containerization",
    )

assert proj.exists
```